components/jms-mq-support/org.wso2.carbon.cache.sync.jms.manager/IBMMQ_README.md

```diff
@@ -0,0 +1,54 @@
+#### Configuring IBM MQ
```

Comment:
Shall we separate the `Configuring IBM MQ` & `Configure the WSO2 IS` into two MDs as in ActiveMQ readmes